Userland/BuggieBox/main.cpp

```diff
@@ -46,12 +46,30 @@ ENUMERATE_UTILITIES(DECLARE_ENTRYPOINT)
 
 static void fail()
 {
-    out(stderr, "Direct running of BuggieBox was detected without finding a proper requested utility.\n");
-    out(stderr, "The following programs are supported: uname, env, lsblk, file, df, mount, umount, mkdir, ");
-    out(stderr, "rmdir, rm, chown, chmod, cp, ln, ls, mv, cat, md5sum, sha1sum, sha256sum, sha512sum, sh, uniq, id, tail, ");
-    out(stderr, "find, less, mknod, ps\n");
-    out(stderr, "To use one of these included utilities, create a symbolic link with the target being this binary, and ensure the basename");
-    out(stderr, "is included within.\n");
+#define TOOL_NAME(name) #name,
+    auto const tool_names = {
+        ENUMERATE_UTILITIES(TOOL_NAME)
+        TOOL_NAME(md5sum) TOOL_NAME(sha1sum) TOOL_NAME(sha256sum) TOOL_NAME(sha512sum)
+        TOOL_NAME(Shell)
+    };
+#undef TOOL_NAME
+
+    outln(stderr);
+    outln(stderr, "Usage:");
+    outln(stderr, "1. Create a symbolic link with the target being this binary,");
+    outln(stderr, "   and ensure the basename is one of the supported utility's name.");
+    outln(stderr, "2. Specify an utility as an argument:");
```

Comment:
"a utility", not "an". English is silly. 😅

Reply:
will fix.